lib/galaxy/files/sources/elabftw.py

```diff
@@ -804,7 +849,7 @@ def _write_from(
         entity_type, entity_id, attachment_id = match.groups()
         entity_type = entity_type.replace("items", "resources")
 
-        return f"elabftw://{location.netloc}/{entity_type}/{entity_id}/{attachment_id}"
+        return f"/{entity_type}/{entity_id}/{attachment_id}"
```

Comment:
Should've been part of galaxyproject/galaxy#19319.

Reply:
Then, is it better to extract that change in its own PR and target the release_24.2 :+1:

Reply:
:+1: [6cdc929](https://github.com/galaxyproject/galaxy/pull/19493/commits/6cdc929716d3779a8e8612cd3ce5ab4f36d6869c)

Reply:
> Then, is it better to extract that change in its own PR and target the release_24.2 👍

galaxyproject/galaxy#19541, not targeting 24.2 because eLabFTW is not part of this version.